src/main/java/views/create_new_month/NewMonthV.java

```diff
@@ -0,0 +1,151 @@
+package views.create_new_month;
+
+import entities.SessionStorage;
+import use_cases.create_new_month.NewMonthOD;
+import use_cases.monthly_menu.MonthMenuOB;
+import use_cases.monthly_menu.UpdateViewIB;
+import use_cases.monthly_menu.UpdateViewUCI;
+import views.load_monthly_menu.LoadMonthMenuVB;
+import views.monthly_menu.MonthMenuP;
+import views.monthly_menu.MonthMenuV;
+import views.monthly_menu.UpdateViewC;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.Objects;
+
+/**
+ * The view class for creating new MonthlyStorage. This class implements the
+ * ActionListener interface. This class calls the controller class to get the
+ * NewMonthOD object, and use the output to set up success and fail view.
+ */
+public class NewMonthV implements ActionListener, LoadMonthMenuVB {
+    NewMonthC controller;
```

Comment:
Shouldn't these all be set to private too? Unless there is a reason they are default, I think we want to give them the most limited scopes if we can.

Reply:
If you are referring to actionPerformed() and loadMonthMenu() they are public because they are implemented from interfaces. 

About the interface containing loadMonthMenu(), I thought Ari would also have this method to open the Month Menu, so I made a interface for it. However, I'm not very sure. @aricasas What do you think?

Reply:
I think they were referring to the instance attributes

Reply:
But regarding loadMonthMenu, I will use it to open the menus, but it doesn't really need to be in an interface. It technically could just be a public method and it still would work fine, but I would say keep it in an interface so that we're very explicit about keeping the method signature a certain way because other classes are calling it

Reply:
Ah, that make sense. I thought Kat was referring to the methods.